docs/docs/development/HOW_TO_START.md

```diff
@@ -1,84 +1,50 @@
 # How to start development
 
-Here's how to get started with contributing to the [Data Center Terraform](https://github.com/atlassian-labs/data-center-terraform) project.
+A number of tools have been used for building this project. When working on the [Data Center Terraform](https://github.com/atlassian-labs/data-center-terraform) project it's recommended that a dev environment is set up with the same tools.
 
-## Requirements
+??? info "CLI Tooling"
 
-Make sure that your development environment is configured with the following tools:
+    A number of CLI tools are recommended for working on this project. See the [Prerequisites guide](../../userguide/PREREQUISITES/) for details.
 
-1. [Terraform](#terraform)
-2. [Helm v3.3 or later](#helm)
-3. [AWS CLI](#aws-cli)
+??? info "Golang"
 
-### Terraform
-
-This project uses Terraform to create and manage the Atlassian Data Center infrastructure on AWS for use with supported Data Center products.
+    [Go](https://go.dev/){.external} is used extensively for testing this project, as such it needs to be installed. Check if Go is already installed by running the following command:
+    
+     ```shell
+     go version
+     ```
+    
+    If Go is not installed, install it by following the [official instructions](https://golang.org/doc/install).
 
-1. Check if Terraform is already installed by running the following command:
+??? info "Pre-commit hook"
 
+    Configure [pre-commit](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks){.external} hook and [TFLint](https://github.com/terraform-linters/tflint){.external} to maintain quality and consistency of the Terraform scripts.
+    
+    Install [pre-commit](https://pre-commit.com/).
+    
     ```shell
-    terraform version
+    brew install pre-commit
     ```
-
-2. If Terraform is not installed, install it by following the [official instructions](https://learn.hashicorp.com/tutorials/terraform/install-cli).
-
-### Helm
-
-Make sure that Helm v3.3 or later is installed on your machine.
-
-1. Check if Helm v3.3 or later is already installed by running the following command:
-
+        
+    In a terminal, change the directory to the repository root and run `pre-commit install`. Now install [TFLint](https://github.com/terraform-linters/tflint).
```

Comment:
We can remove from `Now install [TFLint](httpd://......` until the end of this file.